onnx/test/optimizer_test.py

```diff
@@ -153,6 +153,74 @@ def test_nop_transpose_default(self):
         assert len(list(optimized_model.graph.node)) == 1
         assert optimized_model.graph.node[0].op_type == "Transpose"
 
+    def test_eliminate_unused_initializer(self):
```

Comment:
Nit: add one test case with control flow graph (while/if operator, so you can test subgraph case).